### PR TITLE
add e2e flow to smoketest formatters

### PR DESCRIPTION
**Pull request by bcolloran**
also adds a couple aria roles to make testing a couple of these things easier

### CODE REVIEW

**Comment by bcolloran** [comment]
fixes #3440

File: web-local/test/ui/utils/commonHelpers.ts [hunk 1]
```diff
@@ -85,3 +85,11 @@ export async function waitForValidResource(
     }
   });
 }
+
+export async function waitForDashboard(page: Page) {
```
Comment (ericpgreen2) [inline web-local/test/ui/utils/commonHelpers.ts:89]: `waitForAdbidsDashboard` to be more precise?

**Review by ericpgreen2 — APPROVED** [review]
Looks good!

File: web-local/test/ui/utils/commonHelpers.ts (hunk 1 above)
Comment (bcolloran) [reply web-local/test/ui/utils/commonHelpers.ts]: this line actually fell out of this review when I updated to align with @AdityaHegde's PR. 

agree in principle with this renaming, we can adjust in subsequent PRs. (though i also thought that at the moment the adbids dashboard is the only dashboard we e2e test, to there is no ambiguity?)

**Pull request merged**